app/main.py

```diff
@@ -1,8 +1,24 @@
+from typing import List
+
+
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        Person.people[name] = self
+        self.name = name
+        self.age = age
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons: List[Person] = []
+
+    for person in people:
+        persons.append(Person(person["name"], person["age"]))
+
+    for person in people:
+        if "wife" in person and person["wife"] is not None:
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
+            Person.people[person["wife"]].husband \
+                = Person.people[person["name"]]
```

Comment:
The line continuation with `\` is unnecessary here and can be removed. You can write the assignment in a single line: `Person.people[person["wife"]].husband = Person.people[person["name"]]`.